### PR TITLE
Linter_Bears_Advanced.rst: Fixed indentation

### PR DESCRIPTION
**Pull request by Polcat000**
Fixed indentation on line 123.

Closes https://github.com/coala/coala/issues/5337

### CODE REVIEW

File: docs/Developers/Linter_Bears_Advanced.rst [hunk 1]
```diff
@@ -119,12 +119,12 @@ JSON output is also very common:
 ::
 
     @linter(executable='my_tool')
-    class MyBear:
-    def process_output(self, output, filename, file):
-        for issue in json.loads(output):
-            yield Result.from_values(origin=self,
-                                     message=issue["message"],
-                                     file=filename)
+        class MyBear:
```
Comment (sangamcse) [inline docs/Developers/Linter_Bears_Advanced.rst]: No indentation after `linter` please. This line was fine previously.

File: docs/Developers/Linter_Bears_Advanced.rst [hunk 2]
```diff
@@ -119,12 +119,12 @@ JSON output is also very common:
...
+            def process_output(self, output, filename, file):
+                for issue in json.loads(output):
+                    yield Result.from_values(origin=self,
+                                            message=issue["message"],
```
Comment (sangamcse) [inline docs/Developers/Linter_Bears_Advanced.rst]: match the indentation with `origin=self`

**Comment by sangamcse** [comment]
Please go through the commit guidelines http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html. 
Please use `git commit --amend` and `git push -f` after completing all requested changes. 
Ping me if you are getting any trouble.

**Comment by jayvdb** [comment]
ack https://github.com/coala/coala/pull/5349/commits/c59344a6766cb9582ed489fc092758da235ec7b9

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**